standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/RawStore.java

```diff
@@ -1150,10 +1151,14 @@ Map<String, String> updateTableColumnStatistics(ColumnStatistics colStats, Strin
    * @throws InvalidObjectException the stats object is invalid
    * @throws InvalidInputException unable to record the stats for the table
    */
-  Map<String, String> updatePartitionColumnStatistics(ColumnStatistics statsObj,
+  Map<String, String> updatePartitionColumnStatistics(Table table, MTable mTable, ColumnStatistics statsObj,
```

Comment:
Hi @deniskuzZ,

Thanks for reviewing my PR.
I would like to ask about some of your comments.

- _**You can’t just change the signature for public interface methods.**_ - Then can I create a new method to be able to pass the table arguments?

- **_Also looks like in most places we do not reuse mTable._** - I saw that to get mTable from Table it makes internally calls to HMS. I thought to pass mTable externally too since the goal was to reduce number of calls to HMS for each partition. Can you please explain why it is not good to pass mTable?

-  **_Could you please change only where it’s really needed?_** - I didn’t understand this question, can you please explain what do you mean?

Reply:
signature is fine, github shows like it changed :)